CMakeLists.txt

```diff
@@ -128,7 +133,10 @@ if (CMAKE_USE_PTHREADS_INIT)
     third_party/gtest/googletest
     third_party/gtest/googletest/include
   )
-  add_executable(hexfloat_test ${HEXFLOAT_TEST_SRCS})
+  # TODO(binji): why is this additional test needed for newer CMake?
+  if (BUILD_TESTS)
```

Comment:
Sorry I mistyped my original comment. This is unnecessary, it's line 172 in the current commit that needs to be guarded. It's outside the big existing `if (BUILD_TESTS)`